qemu/tests/qemu_disk_img_convert.py

```diff
@@ -33,6 +36,28 @@ def convert(self, t_params=None):
         self.trash.append(converted)
         return params
 
+    @error_context.context_aware
```

Comment:
When we test strict mode, we just use it to compare with normal mode. We don't have test scenario which only test strict mode.